src/Servers/HttpSys/src/NativeInterop/RequestQueue.cs

```diff
@@ -39,18 +74,30 @@ internal RequestQueue(UrlGroup urlGroup, string requestQueueName, ILogger logger
                     UnsafeNclNativeMethods.FileCompletionNotificationModes.SkipCompletionPortOnSuccess |
                     UnsafeNclNativeMethods.FileCompletionNotificationModes.SkipSetEventOnHandle))
             {
+                requestQueueHandle.Dispose();
```

Comment:
Why was this added?

Reply:
Eager clean up. If this secondary API call fails then the requestQueueHandle would only be cleaned up by its finalizer. The process is probably going to shut down anyways, but it's best to clean things up as locally as possible.